selenium exercises/ex12.py

```diff
@@ -0,0 +1,50 @@
+from login_module import LoginModule
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from time import sleep
+from selenium.common.exceptions import WebDriverException
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver import Chrome
+from time import sleep
+from selenium.webdriver.chrome.service import Service as chromeService
+
+driver_path = "C:/Users/91989/Downloads/driver/chromedriver.exe"
+url = 'https://www.saucedemo.com/v1/'
+
+chrome_service = chromeService(executable_path=driver_path)
+driver=Chrome(service=chrome_service)
+
+username_id='user-name'
+password_id='password'
+login_id='login-button'
+product1_id='//*[@id="inventory_container"]/div/div[1]/div[3]/button'
+product2_id='//*[@id="inventory_container"]/div/div[2]/div[3]/button'
```

Comment:
I dont think these are ids, looks like xpaths, provide proper name to avoid confusion.